tests/integration/integration_test.py

```diff
@@ -0,0 +1,87 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from marquez_client.client import Client
+
+import logging
+import os
+import pytest
+import requests
+import subprocess
+import sys
+from urllib3.util.retry import Retry
+
+
+def test_data_in_marquez():
+
+    dag_id = "test_dag_v2"
+    execution_date = "2019-02-01T00:00:00"
+    namespace = "integration-test"
+
+    c = Client(namespace_name=namespace)
+    wait_for_marquez()
+    assert(init_airflow_db())
+    assert(trigger_dag(dag_id, execution_date))
+    assert(check_dag_state(dag_id, execution_date))
+
+    try:
+        result = c.get_namespace(namespace)
+    except Exception as e:
```

Comment:
`get_namespace(..)` will return empty on a 404, not an exception.

Reply:
There are two other exception types that can be thrown, so I'd suggest catching them explicitly instead of using a catch-all.

Reply:
Ok, I removed the catch for exceptions.